imap_processing/cdf/utils.py

```diff
@@ -76,8 +77,8 @@ def load_cdf(
     """
     # TODO: remove this when cdflib is updated to version >1.3.0
     if "to_datetime" not in kwargs:
-        kwargs["to_datetime"] = True
-    dataset = cdf_to_xarray(file_path, kwargs)
+        to_datetime_arg = kwargs.get("to_datetime", True)
```

Comment:
I think that this needs to be pop to avoid a duplicate `to_datetime` keyword. Since this code is soon to be removed, I wouldn't sweat this too much.
```suggestion
        to_datetime_arg = kwargs.pop("to_datetime", True)
```